include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -153,9 +153,18 @@ struct base_turn_handler
             typename UmbrellaStrategy,
             typename TurnInfo
     >
-    static inline void both_collinear(UniqueSubRange1 const& range_p,
-            UniqueSubRange2 const& range_q,
-            UmbrellaStrategy const& umbrella_strategy,
+    static inline void both_collinear(
+            UniqueSubRange1 const&
+#if ! defined(BOOST_GEOMETRY_USE_RESCALING)
+            range_p
+#endif
+            ,
+            UniqueSubRange2 const&
+#if ! defined(BOOST_GEOMETRY_USE_RESCALING)
+            range_q
+#endif
```

Comment:
I think I'd prefer passing them to `ignore_unused` to avoid ifdefs distorting reading of the code.

Reply:
Thanks, I agree.

Reply:
Thank you, I agree with that too.
Furthermore I'm OK, thanks for the PR